diff --git a/net/minecraft/server/GuiLogFormatter.java b/net/minecraft/server/GuiLogFormatter.java
index ced8777..5b294db 100644
--- a/net/minecraft/server/GuiLogFormatter.java
+++ b/net/minecraft/server/GuiLogFormatter.java
@@ -17,8 +17,8 @@ class GuiLogFormatter extends Formatter {
     final GuiLogOutputHandler a; /* synthetic field */
 
     GuiLogFormatter(GuiLogOutputHandler guilogoutputhandler) {
-        a = guilogoutputhandler;
         super();
+        a = guilogoutputhandler;
     }
 
     public String format(LogRecord logrecord) {
diff --git a/net/minecraft/server/GuiStatsListener.java b/net/minecraft/server/GuiStatsListener.java
index 26b87a3..b37ed99 100644
--- a/net/minecraft/server/GuiStatsListener.java
+++ b/net/minecraft/server/GuiStatsListener.java
@@ -17,8 +17,8 @@ class GuiStatsListener
     final GuiStatsComponent a; /* synthetic field */
 
     GuiStatsListener(GuiStatsComponent guistatscomponent) {
-        a = guistatscomponent;
         super();
+        a = guistatscomponent;
     }
 
     public void actionPerformed(ActionEvent actionevent) {
diff --git a/net/minecraft/server/MinecartTrackLogic.java b/net/minecraft/server/MinecartTrackLogic.java
index d0430f7..886dcae 100644
--- a/net/minecraft/server/MinecartTrackLogic.java
+++ b/net/minecraft/server/MinecartTrackLogic.java
@@ -22,8 +22,8 @@ class MinecartTrackLogic {
     final BlockMinecartTrack a; /* synthetic field */
 
     public MinecartTrackLogic(BlockMinecartTrack blockminecarttrack, World world, int i, int j, int k) {
-        a = blockminecarttrack;
         super();
+        a = blockminecarttrack;
         g = ((List) (new ArrayList()));
         b = world;
         c = i;
diff --git a/net/minecraft/server/NetworkAcceptThread.java b/net/minecraft/server/NetworkAcceptThread.java
index 8fef705..cd5e19f 100644
--- a/net/minecraft/server/NetworkAcceptThread.java
+++ b/net/minecraft/server/NetworkAcceptThread.java
@@ -17,9 +17,9 @@ class NetworkAcceptThread extends Thread {
     final NetworkListenThread b; /* synthetic field */
 
     NetworkAcceptThread(NetworkListenThread networklistenthread, String s, MinecraftServer minecraftserver) {
+        super(s);
         b = networklistenthread;
         a = minecraftserver;
-        super(s);
     }
 
     public void run() {
diff --git a/net/minecraft/server/NetworkMasterThread.java b/net/minecraft/server/NetworkMasterThread.java
index b004ab9..eee7ad2 100644
--- a/net/minecraft/server/NetworkMasterThread.java
+++ b/net/minecraft/server/NetworkMasterThread.java
@@ -14,8 +14,8 @@ class NetworkMasterThread extends Thread {
     final NetworkManager a; /* synthetic field */
 
     NetworkMasterThread(NetworkManager networkmanager) {
-        a = networkmanager;
         super();
+        a = networkmanager;
     }
 
     public void run() {
diff --git a/net/minecraft/server/NetworkReaderThread.java b/net/minecraft/server/NetworkReaderThread.java
index bd26eda..0243371 100644
--- a/net/minecraft/server/NetworkReaderThread.java
+++ b/net/minecraft/server/NetworkReaderThread.java
@@ -14,24 +14,20 @@ class NetworkReaderThread extends Thread {
     final NetworkManager a; /* synthetic field */
 
     NetworkReaderThread(NetworkManager networkmanager, String s) {
-        a = networkmanager;
         super(s);
+        a = networkmanager;
     }
 
     public void run() {
         synchronized(NetworkManager.a) {
             NetworkManager.b++;
         }
-        for(; NetworkManager.a(a) && !NetworkManager.b(a); NetworkManager.c(a));
-        synchronized(NetworkManager.a) {
-            NetworkManager.b--;
-        }
-        break MISSING_BLOCK_LABEL_115;
-        Exception exception2;
-        exception2;
-        synchronized(NetworkManager.a) {
-            NetworkManager.b--;
+        try {
+            for(; NetworkManager.a(a) && !NetworkManager.b(a); NetworkManager.c(a));
+        } finally {
+            synchronized(NetworkManager.a) {
+                NetworkManager.b--;
+            }
         }
-        throw exception2;
     }
 }
diff --git a/net/minecraft/server/NetworkWriterThread.java b/net/minecraft/server/NetworkWriterThread.java
index 2da0529..618ddb5 100644
--- a/net/minecraft/server/NetworkWriterThread.java
+++ b/net/minecraft/server/NetworkWriterThread.java
@@ -14,24 +14,20 @@ class NetworkWriterThread extends Thread {
     final NetworkManager a; /* synthetic field */
 
     NetworkWriterThread(NetworkManager networkmanager, String s) {
-        a = networkmanager;
         super(s);
+        a = networkmanager;
     }
 
     public void run() {
         synchronized(NetworkManager.a) {
             NetworkManager.c++;
         }
-        for(; NetworkManager.a(a); NetworkManager.d(a));
-        synchronized(NetworkManager.a) {
-            NetworkManager.c--;
-        }
-        break MISSING_BLOCK_LABEL_105;
-        Exception exception2;
-        exception2;
-        synchronized(NetworkManager.a) {
-            NetworkManager.c--;
+        try {
+            for(; NetworkManager.a(a); NetworkManager.d(a));
+        } finally {
+            synchronized(NetworkManager.a) {
+                NetworkManager.c--;
+            }
         }
-        throw exception2;
     }
 }
diff --git a/net/minecraft/server/PlayerInstance.java b/net/minecraft/server/PlayerInstance.java
index d87058d..dcce686 100644
--- a/net/minecraft/server/PlayerInstance.java
+++ b/net/minecraft/server/PlayerInstance.java
@@ -31,8 +31,8 @@ class PlayerInstance {
     final PlayerManager a; /* synthetic field */
 
     public PlayerInstance(PlayerManager playermanager, int i1, int j1) {
-        a = playermanager;
         super();
+        a = playermanager;
         b = ((List) (new ArrayList()));
         f = new short[10];
         g = 0;
diff --git a/net/minecraft/server/RecipeSorter.java b/net/minecraft/server/RecipeSorter.java
index 23a0dd2..1cc6561 100644
--- a/net/minecraft/server/RecipeSorter.java
+++ b/net/minecraft/server/RecipeSorter.java
@@ -16,8 +16,8 @@ class RecipeSorter
     final CraftingManager a; /* synthetic field */
 
     RecipeSorter(CraftingManager craftingmanager) {
-        a = craftingmanager;
         super();
+        a = craftingmanager;
     }
 
     public int a(CraftingRecipe craftingrecipe, CraftingRecipe craftingrecipe1) {
diff --git a/net/minecraft/server/ServerGuiCommandListener.java b/net/minecraft/server/ServerGuiCommandListener.java
index 17f9866..261c2c7 100644
--- a/net/minecraft/server/ServerGuiCommandListener.java
+++ b/net/minecraft/server/ServerGuiCommandListener.java
@@ -19,9 +19,9 @@ class ServerGuiCommandListener
     final ServerGUI b; /* synthetic field */
 
     ServerGuiCommandListener(ServerGUI servergui, JTextField jtextfield) {
+        super();
         b = servergui;
         a = jtextfield;
-        super();
     }
 
     public void actionPerformed(ActionEvent actionevent) {
diff --git a/net/minecraft/server/ServerGuiFocusAdapter.java b/net/minecraft/server/ServerGuiFocusAdapter.java
index f484d52..6e95b0a 100644
--- a/net/minecraft/server/ServerGuiFocusAdapter.java
+++ b/net/minecraft/server/ServerGuiFocusAdapter.java
@@ -16,8 +16,8 @@ class ServerGuiFocusAdapter extends FocusAdapter {
     final ServerGUI a; /* synthetic field */
 
     ServerGuiFocusAdapter(ServerGUI servergui) {
-        a = servergui;
         super();
+        a = servergui;
     }
 
     public void focusGained(FocusEvent focusevent) {
diff --git a/net/minecraft/server/ServerWindowAdapter.java b/net/minecraft/server/ServerWindowAdapter.java
index 1c1b29c..9d582ea 100644
--- a/net/minecraft/server/ServerWindowAdapter.java
+++ b/net/minecraft/server/ServerWindowAdapter.java
@@ -16,8 +16,8 @@ final class ServerWindowAdapter extends WindowAdapter {
     final MinecraftServer a; /* synthetic field */
 
     ServerWindowAdapter(MinecraftServer minecraftserver) {
-        a = minecraftserver;
         super();
+        a = minecraftserver;
     }
 
     public void windowClosing(WindowEvent windowevent) {
diff --git a/net/minecraft/server/SlotArmor.java b/net/minecraft/server/SlotArmor.java
index dd40d9a..f4b1e10 100644
--- a/net/minecraft/server/SlotArmor.java
+++ b/net/minecraft/server/SlotArmor.java
@@ -16,9 +16,9 @@ class SlotArmor extends Slot {
     final CraftingInventoryPlayerCB b; /* synthetic field */
 
     SlotArmor(CraftingInventoryPlayerCB craftinginventoryplayercb, IInventory iinventory, int i, int j, int k, int l) {
+        super(iinventory, i, j, k);
         b = craftinginventoryplayercb;
         a = l;
-        super(iinventory, i, j, k);
     }
 
     public int a() {
diff --git a/net/minecraft/server/SpawnerAnimals.java b/net/minecraft/server/SpawnerAnimals.java
index 471c49b..69a08a2 100644
--- a/net/minecraft/server/SpawnerAnimals.java
+++ b/net/minecraft/server/SpawnerAnimals.java
@@ -28,127 +28,100 @@ public final class SpawnerAnimals {
     }
 
     public static final int a(World world) {
-        int j;
-        int k;
         a.clear();
-        for(int i = 0; i < world.d.size(); i++) {
+
+        for(int i = 0; i < world.d.size(); ++i) {
             EntityPlayer entityplayer = (EntityPlayer)world.d.get(i);
-            int l = MathHelper.b(entityplayer.p / 16D);
-            int i1 = MathHelper.b(entityplayer.r / 16D);
+            int l = MathHelper.b(entityplayer.p / 16.0D);
+            int i1 = MathHelper.b(entityplayer.r / 16.0D);
             byte byte0 = 8;
-            for(int j1 = -byte0; j1 <= byte0; j1++) {
-                for(int k1 = -byte0; k1 <= byte0; k1++)
-                    a.add(((Object) (new ChunkCoordIntPair(j1 + l, k1 + i1))));
 
+            for(int j = -byte0; j <= byte0; ++j) {
+                for(int k = -byte0; k <= byte0; ++k) {
+                    a.add(new ChunkCoordIntPair(j + l, k + i1));
+                }
             }
-
         }
 
-        j = 0;
-        k = 0;
-_L10:
-        if(k >= EnumCreatureType.values().length) goto _L2; else goto _L1
-_L1:
-        EnumCreatureType enumcreaturetype;
-        Iterator iterator;
-        enumcreaturetype = EnumCreatureType.values()[k];
-        if(world.a(enumcreaturetype.c) > (enumcreaturetype.d * a.size()) / 256)
-            continue; /* Loop/switch isn't completed */
-        iterator = a.iterator();
-_L4:
-        Class aclass[];
-        int l1;
-        int i2;
-        int j2;
-        int k2;
-        int l2;
-        int i3;
-        do {
-            ChunkCoordIntPair chunkcoordintpair;
-            do {
-                do {
-                    if(!iterator.hasNext())
-                        continue; /* Loop/switch isn't completed */
-                    chunkcoordintpair = (ChunkCoordIntPair)iterator.next();
-                } while(world.l.nextInt(50) != 0);
-                MobSpawnerBase mobspawnerbase = world.a().a(chunkcoordintpair);
-                aclass = mobspawnerbase.a(enumcreaturetype);
-            } while(aclass == null || aclass.length == 0);
-            l1 = world.l.nextInt(aclass.length);
-            ChunkPosition chunkposition = a(world, chunkcoordintpair.a * 16, chunkcoordintpair.b * 16);
-            i2 = chunkposition.a;
-            j2 = chunkposition.b;
-            k2 = chunkposition.c;
-        } while(world.d(i2, j2, k2) || world.c(i2, j2, k2) != Material.a);
-        l2 = 0;
-        i3 = 0;
-_L9:
-        if(i3 >= 3) goto _L4; else goto _L3
-_L3:
-        int j3;
-        int k3;
-        int l3;
-        byte byte1;
-        int i4;
-        j3 = i2;
-        k3 = j2;
-        l3 = k2;
-        byte1 = 6;
-        i4 = 0;
-_L8:
-        if(i4 >= 4) goto _L6; else goto _L5
-_L5:
-        EntityLiving entityliving;
-        j3 += world.l.nextInt(((int) (byte1))) - world.l.nextInt(((int) (byte1)));
-        k3 += world.l.nextInt(1) - world.l.nextInt(1);
-        l3 += world.l.nextInt(((int) (byte1))) - world.l.nextInt(((int) (byte1)));
-        if(!world.d(j3, k3 - 1, l3) || world.d(j3, k3, l3) || world.c(j3, k3, l3).d() || world.d(j3, k3 + 1, l3))
-            continue; /* Loop/switch isn't completed */
-        float f = (float)j3 + 0.5F;
-        float f1 = k3;
-        float f2 = (float)l3 + 0.5F;
-        if(world.a(f, f1, f2, 24D) != null)
-            continue; /* Loop/switch isn't completed */
-        float f3 = f - (float)world.m;
-        float f4 = f1 - (float)world.n;
-        float f5 = f2 - (float)world.o;
-        float f6 = f3 * f3 + f4 * f4 + f5 * f5;
-        if(f6 < 576F)
-            continue; /* Loop/switch isn't completed */
-        try {
-            entityliving = (EntityLiving)aclass[l1].getConstructor(new Class[] {
-                net.minecraft.server.World.class
-            }).newInstance(new Object[] {
-                world
-            });
-        }
-        catch(Exception exception) {
-            exception.printStackTrace();
-            return j;
-        }
-        entityliving.c(f, f1, f2, world.l.nextFloat() * 360F, 0.0F);
-        if(!entityliving.a())
-            break; /* Loop/switch isn't completed */
-        l2++;
-        world.a(((Entity) (entityliving)));
-        if((entityliving instanceof EntitySpider) && world.l.nextInt(100) == 0) {
-            EntitySkeleton entityskeleton = new EntitySkeleton(world);
-            entityskeleton.c(f, f1, f2, entityliving.v, 0.0F);
-            world.a(((Entity) (entityskeleton)));
-            entityskeleton.e(((Entity) (entityliving)));
+        int i = 0;
+
+        for(int k = 0; k < EnumCreatureType.values().length; ++k) {
+            EnumCreatureType enumcreaturetype = EnumCreatureType.values()[k];
+            if(world.a(enumcreaturetype.c) <= enumcreaturetype.d * a.size() / 256) {
+                Iterator iterator = a.iterator();
+
+                label90:
+                while(iterator.hasNext()) {
+                    ChunkCoordIntPair chunkcoordintpair = (ChunkCoordIntPair)iterator.next();
+                    if(world.l.nextInt(50) == 0) {
+                        MobSpawnerBase mobspawnerbase = world.a().a(chunkcoordintpair);
+                        Class[] aclass = mobspawnerbase.a(enumcreaturetype);
+                        if(aclass != null && aclass.length != 0) {
+                            int l1 = world.l.nextInt(aclass.length);
+                            ChunkPosition chunkposition = a(world, chunkcoordintpair.a * 16, chunkcoordintpair.b * 16);
+                            int i1 = chunkposition.a;
+                            int j1 = chunkposition.b;
+                            int k1 = chunkposition.c;
+                            if(!world.d(i1, j1, k1) && world.c(i1, j1, k1) == Material.a) {
+                                int j = 0;
+
+                                for(int l2 = 0; l2 < 3; ++l2) {
+                                    int i2 = i1;
+                                    int j2 = j1;
+                                    int k2 = k1;
+                                    byte byte0 = 6;
+
+                                    for(int l3 = 0; l3 < 4; ++l3) {
+                                        i2 += world.l.nextInt(byte0) - world.l.nextInt(byte0);
+                                        j2 += world.l.nextInt(1) - world.l.nextInt(1);
+                                        k2 += world.l.nextInt(byte0) - world.l.nextInt(byte0);
+                                        if(world.d(i2, j2 - 1, k2) && !world.d(i2, j2, k2) && !world.c(i2, j2, k2).d() && !world.d(i2, j2 + 1, k2)) {
+                                            float f = (float)i2 + 0.5F;
+                                            float f1 = (float)j2;
+                                            float f2 = (float)k2 + 0.5F;
+                                            if(world.a((double)f, (double)f1, (double)f2, 24.0D) == null) {
+                                                float f3 = f - (float)world.m;
+                                                float f4 = f1 - (float)world.n;
+                                                float f5 = f2 - (float)world.o;
+                                                float f6 = f3 * f3 + f4 * f4 + f5 * f5;
+                                                if(f6 >= 576.0F) {
+                                                    EntityLiving entityliving;
+                                                    try {
+                                                        entityliving = (EntityLiving)aclass[l1].getConstructor(new Class[]{World.class}).newInstance(new Object[]{world});
+                                                    } catch (Exception exception) {
+                                                        exception.printStackTrace();
+                                                        return i;
+                                                    }
+
+                                                    entityliving.c((double)f, (double)f1, (double)f2, world.l.nextFloat() * 360.0F, 0.0F);
+                                                    if(entityliving.a()) {
+                                                        ++j;
+                                                        world.a(entityliving);
+                                                        if(entityliving instanceof EntitySpider && world.l.nextInt(100) == 0) {
+                                                            EntitySkeleton entityskeleton = new EntitySkeleton(world);
+                                                            entityskeleton.c((double)f, (double)f1, (double)f2, entityliving.v, 0.0F);
+                                                            world.a(entityskeleton);
+                                                            entityskeleton.e(entityliving);
+                                                        }
+
+                                                        if(j >= entityliving.i()) {
+                                                            continue label90;
+                                                        }
+                                                    }
+
+                                                    i += j;
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
         }
-        if(l2 < entityliving.i()) goto _L7; else goto _L4
-_L7:
-        j += l2;
-        i4++;
-          goto _L8
-_L6:
-        i3++;
-          goto _L9
-        k++;
-          goto _L10
-_L2:
-        return j;
+        return i;
     }
 
 }
diff --git a/net/minecraft/server/ThreadCommandReader.java b/net/minecraft/server/ThreadCommandReader.java
index 1b5ef16..0698b07 100644
--- a/net/minecraft/server/ThreadCommandReader.java
+++ b/net/minecraft/server/ThreadCommandReader.java
@@ -15,8 +15,8 @@ public class ThreadCommandReader extends Thread {
     final MinecraftServer a; /* synthetic field */
 
     public ThreadCommandReader(MinecraftServer minecraftserver) {
-        a = minecraftserver;
         super();
+        a = minecraftserver;
     }
 
     public void run() {
diff --git a/net/minecraft/server/ThreadLoginVerifier.java b/net/minecraft/server/ThreadLoginVerifier.java
index 5c6e0c5..9ef8d85 100644
--- a/net/minecraft/server/ThreadLoginVerifier.java
+++ b/net/minecraft/server/ThreadLoginVerifier.java
@@ -18,9 +18,9 @@ class ThreadLoginVerifier extends Thread {
     final NetLoginHandler b; /* synthetic field */
 
     ThreadLoginVerifier(NetLoginHandler netloginhandler, Packet1Login packet1login) {
+        super();
         b = netloginhandler;
         a = packet1login;
-        super();
     }
 
     public void run() {
diff --git a/net/minecraft/server/ThreadMonitorConnection.java b/net/minecraft/server/ThreadMonitorConnection.java
index c8e195c..1e897c9 100644
--- a/net/minecraft/server/ThreadMonitorConnection.java
+++ b/net/minecraft/server/ThreadMonitorConnection.java
@@ -14,8 +14,8 @@ class ThreadMonitorConnection extends Thread {
     final NetworkManager a; /* synthetic field */
 
     ThreadMonitorConnection(NetworkManager networkmanager) {
-        a = networkmanager;
         super();
+        a = networkmanager;
     }
 
     public void run() {
diff --git a/net/minecraft/server/ThreadServerApplication.java b/net/minecraft/server/ThreadServerApplication.java
index e97d224..9a81f64 100644
--- a/net/minecraft/server/ThreadServerApplication.java
+++ b/net/minecraft/server/ThreadServerApplication.java
@@ -14,8 +14,8 @@ public final class ThreadServerApplication extends Thread {
     final MinecraftServer a; /* synthetic field */
 
     public ThreadServerApplication(String s, MinecraftServer minecraftserver) {
-        a = minecraftserver;
         super(s);
+        a = minecraftserver;
     }
 
     public void run() {
diff --git a/net/minecraft/server/ThreadSleepForever.java b/net/minecraft/server/ThreadSleepForever.java
index 8e77865..01db0e0 100644
--- a/net/minecraft/server/ThreadSleepForever.java
+++ b/net/minecraft/server/ThreadSleepForever.java
@@ -14,8 +14,8 @@ public class ThreadSleepForever extends Thread {
     final MinecraftServer a; /* synthetic field */
 
     public ThreadSleepForever(MinecraftServer minecraftserver) {
-        a = minecraftserver;
         super();
+        a = minecraftserver;
         setDaemon(true);
         start();
     }
